aiohue/v2/controllers/groups.py

```diff
@@ -142,6 +144,27 @@ async def set_state(
 
         await self.update(id, update_obj)
 
+    async def set_dimming_delta(
+        self, id: str, brightness_delta: float | None = None
+    ) -> None:
+        """
+        Set brightness_delta value and action via DimmingDeltaFeature.
+
+        The action to be send depends on brightness_delta value:
+         > 0: UP,
+         < 0: DOWN,
+         else: STOP (this immediately stops any dimming transition)
+        """
+        if brightness_delta is not None:
+            update_obj = GroupedLightPut()
+            action = DeltaAction.DOWN if brightness_delta < 0 else DeltaAction.UP if brightness_delta > 0 else DeltaAction.STOP
```

Comment:
please split this one up into a specific definition for the None case 

```
if brightness_delta is None or brightness_delta == 0:
    action = DeltaAction.STOP
elif brightness_delta < 0:
  ...
else:
    ...

Reply:
done